code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -681,11 +686,27 @@ GLOBAL_VAR(bomb_set)
 	/// Is this a training disk?
 	var/training = FALSE
 
+/obj/item/disk/nuclear/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>You should keep this safe...</span>"
```

Comment:
Danger spans shouldn't be used outside of combat or something happening to the user really. Our descriptions are *mostly* standardized to use only notice and warning.
```suggestion
	. += "<span class='warning'>You should keep this safe...</span>"
```